### PR TITLE
support sampler and operation ignore config

### PR DESCRIPTION
**Pull request by wangrzneu**

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please submit issues to discuss or include full document in the pull request.

**Comment by wangrzneu** [comment]
> Please submit issues to discuss or include full document in the pull request.

Issue (https://github.com/apache/skywalking/issues/7938) has been submitted, and I will add some integration tests before review.

File: examples/nginx.conf [hunk 1]
```diff
@@ -25,7 +25,7 @@ events {
     worker_connections 1024;
 }
 http {
-    lua_package_path "/Users/wusheng/Documents/GitHub/skywalking-nginx-lua/lib/?.lua;;";
+    lua_package_path "/Users/user/Workspace/skywalking-nginx-lua/lib/?.lua;;";
```
Comment (wu-sheng) [inline examples/nginx.conf]: This makes no difference. No matter you pointing to, another developer could not have it.

**Comment by wu-sheng** [comment]
No update here, and recommend to follow this, https://github.com/apache/skywalking/issues/7938#issuecomment-962611461

**Pull request closed without merging**